### DISCUSSION TITLE
Date time for Sanoid and Syncoid - How to Change

### DISCUSSION THREAD

**Discussion by masterwishx** [jimsalterjrs/sanoid#833] [Q&A]
appdata@autosnap_2023-07-21_10:21:13_daily 
system_cache_ssd/appdata@syncoid_MYServer_2023-07-21:10:21:15-GMT03:00

These are Synoid and Syncoid snapshots how to change date time to d-m-Y_H:i:s ?

**Comment by phreaker0** [comment]
this is currently not supported

**Reply by masterwishx** [reply]
Sorry, hoping will be suport in future ...
